.bazelrc

```diff
@@ -45,6 +45,6 @@ build --noenable_bzlmod
 # feature flags
 #
 
-# value [auto, on, off]
-# 'auto' is platform dependent ('on' on Linux and QNX, 'off' on other OS) and the default value if the flag is not set
-#build --//:feature_acl=off
+# value [True, False]. Default: False.
+# Here we turn on the setting for Linux in this repo. Downstream consumers are unaffected.
+build:linux --//:feature_acl=True
```

Comment:
I'm not sure I like this approach. It changes the semantics since this file is only respected for the iceoryx repository. For our users that means that ACL is now suddenly deactivated by default. What's wrong with the previous approach?

Reply:
As mentioned in the ticket, the previous approach caused broken builds in CIs such as ours and the Bazel BCR where libacl is not installed on the host (and should not be).

As you mentioned that Bazel is only really supported from the upcoming 3.0 version I feel this default isn't set in stone, wdyt?

Reply:
It's not set in stone, but since bazel was not supported in v2.0, quite a lot of our users which need bazel are already using v2.95. I know especially about users who are still using the WORKSPACE approach with bazel 6.2, that's why I'm more concerned with that one and I think we didn't silently break anything for them with the recent PRs. This one would be a silent break since libacl would not be used anymore by default.

Reply:
It could be added to the change log / release notes and then it's not as silent anymore ;)